.github/workflows/ci.yml

```diff
@@ -0,0 +1,114 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
This is fine, but it will (afaik) run the job twice when you push to a PR: once for the push, once for the PR synchronized event.

A lot of people will do:

```yaml
on:
  push:
    branches:
      - master
      - etc.
  pull_request:
jobs:
  ...
```

It's not a big deal; up to you.

Reply:
Ah gotcha thanks, will fix.